spec/controllers/mailings_controller_spec.rb

```diff
@@ -6,17 +6,14 @@
   let(:mailing)        { FactoryGirl.create(:mailing) }
   let(:user)           { FactoryGirl.create(:user) }
 
-  shared_context 'User logged in' do
-    before { sign_in user }
-  end
-
   describe 'GET index' do
     before { mailing }
 
     context 'with user logged in' do
-      include_context 'User logged in'
+      # include_context 'with user logged in' # => red
+      before { sign_in user }                 # => green
```

Comment:
If you want to have the call to `index` dryed out of the examples, you could also do
```ruby
before do
  sign_in user
  get :index
end
```